### DISCUSSION TITLE
Is there any way to make incoming direction and outgoing direction of a stop point in holonomic mode other than 0 or 180 degrees?

### DISCUSSION THREAD

**Discussion by richardthedou** [mjansen4857/pathplanner#201] [Ideas]
In path planner, when using the holonomic mode, it seems that the incoming direction and outgoing direction of a stop point either have to be the same or reversed (when selecting the "reversal" option). However, holonomic drive trains like Swerve can arrive and stop at a point from 0 degrees and exit at 90 degrees for example. Is there any way to implement this kind of path on PP? If not, I think that this will be a great feature to add. Thanks!

**Comment by mjansen4857** [comment]
#163 

Not working on this for now since it would cause some issues. Maybe next year but it is too late for me to be comfortable with it for 2023.

**Reply by mjansen4857** [reply]
Actually, I forgot how my own software works so after thinking about it some more it should be fairly easy to do without breaking anything. I'll look into it

**Reply by mjansen4857** [reply]
Playing around with it, it will need some fairly significant UI changes to implement properly. So, I will be waiting for 2024. It will be a lot nicer with the new UI stuff I have planned for next year instead of trying to squeeze it in to the existing stuff in less than a week.

**Reply by richardthedou** [reply]
Got it! That sounds legit. After thinking about it, it seems that concatenating two paths will have the same effect, so it shouldn't be a big deal.
